internal/app/machined/pkg/controllers/network/operator/utils/utils.go

```diff
@@ -0,0 +1,22 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package utils provides common methods for operators.
+package utils
+
+import (
+	"fmt"
+	"net"
+	"net/netip"
+)
+
+// ToUDPAddr combines the given net.IP and port to form a net.UDPAddr.
+func ToUDPAddr(ip net.IP, port uint16) (*net.UDPAddr, error) {
```

Comment:
move to `internal/` under `operator/` ? 

move `resources/network`?

Reply:
ended up removing this file and doing a straightforward `&net.UDPAddr{}` initialization